scripts/parse.sh

```diff
@@ -28,15 +28,20 @@ python3 scripts/combine.py
 python3 scripts/reformat.py
 
 # Copy canteens.json in the output directory:
-echo "Coppying canteens..."
+echo "Copying canteens..."
 cp src/canteens.json $OUT_DIR
 echo "Done"
 
 openmensa_list=( "ipp-bistro" "fmi-bistro" )
 
-for loc in "${openmensa_list[@]}"; do
-    echo "Parsing openmensa menus for: " "$loc"
-    python3 src/main.py -p "$loc" --openmensa "./dist/$loc"
+for canteen in "${openmensa_list[@]}"; do
+    echo "Parsing openmensa menus for: " "$canteen"
+    python3 src/main.py -p "$canteen" --openmensa "./dist/$canteen"
 done
 
+ENUM_JSON_PATH="./dist/enums"
```

Comment:
Here again, the OUT_DIR variable is probably better

Reply:
ok